.github/CODEOWNERS_baseline_errors.txt

```diff
@@ -281,4 +280,4 @@ tmahmood-microsoft is not a public member of Azure.
 Aviv-Yaniv is not a public member of Azure.
 sakash279 is an invalid user. Ensure the user exists, is public member of Azure and has write permissions.
 sivethe is an invalid user. Ensure the user exists, is public member of Azure and has write permissions.
-ThomasWeiss is an invalid user. Ensure the user exists, is public member of Azure and has write permissions.
\ No newline at end of file
+ThomasWeiss is an invalid user. Ensure the user exists, is public member of Azure and has write permissions.
```

Comment:
Is this just a line endings change?

Reply:
I didn't explicitly do that, so I'm guessing so.    This was a direct edit on GH, so they probably did an automatic unix-style swap.